languages/ruby/bitwarden_sdk_secrets/sig/bitwarden-sdk.rbs

```diff
@@ -8,6 +8,6 @@ class BitwardenClient
   attr_reader secrets_client: SecretsClient
```

Comment:
I though we changed this to `projects`, `secrets` with additional `auth` ?

Reply:
Addressed in 83650b9c and ab17aa36.